complete/src/main/java/hello/Application.java

```diff
@@ -13,7 +13,10 @@
 public class Application extends AsyncConfigurerSupport {
 
     public static void main(String[] args) {
-        SpringApplication.run(Application.class, args);
+        SpringApplication.run(Application.class, args).close();
+
+        // Close application
+        System.exit(0);
     }
```

Comment:
have changed it to `@Bean`. Left the customisation of `Executor` there since more information is there regarding this in the guide. @gregturn @dsyer let me know if you think I should make more changes